txt2tags.py

```diff
@@ -272,7 +272,7 @@
 <h3>%(HEADER3)s</h3>
 </hgroup>
 </header>
-<article>
+
```

Comment:
Why not use "article"?

Reply:
Validator shows warning:
-----------------------------------------
Warning: Article lacks heading. Consider using h2-h6 elements to add identifying headings to all articles.
From line 37, column 1; to line 37, column 9
</header>↩<article>↩↩<div
------------------------------------------
from https://dev.w3.org/html5/html-author/#the-article-element:
-------------------------------------------
4.3.4.4 The article element
The article element represents an independent section of a document, page, or site. This could be a forum post, a magazine or newspaper article, a blog entry, a user-submitted comment, or any other independent item of content.
--------------------------------------------
If the converted html page is a standalone document, it may contain an article but may not completely only an article.